### PR TITLE
Add team admins/managers to contacts list.

### PR DESCRIPTION
**Pull request by chillburn**
I've added team admins/managers to the contacts list. 
I was going to remove teamManagers with user_id that exist in the contacts already but i need a team to test with. 

Team Tommy seems to have bad data. 

user_id
36197 | Kam Low
36226 | Kam Bam
1 | Kam Bam

### CODE REVIEW

File: src/store/modules/contacts/getters.js [hunk 1]
```diff
@@ -41,9 +41,11 @@ const getters = {
 
   filteredContactsSorted: (state, getters, rootState, rootGetters) => (excludeUserId = 0) => {
     if (!state.contacts.length) return [];
-
-    const contacts = state.contacts.filter(x => x.friend_id !== rootState.user.id && ['active', 'accepted'].includes(x.status));
-
+    const filteredContacts = state.contacts.filter(x => x.friend_id !== rootState.user.id && ['active', 'accepted'].includes(x.status));
+    const teamManagers = rootState.team.teamMembers.filter(x => {
```
Comment (auscaster) [inline src/store/modules/contacts/getters.js:45]: Why only managers? Should be able to create a chat with any team member that exists the team members array. Also need to remove duplicates if already in contacts list.

**Review by auscaster — CHANGES_REQUESTED** [review]

**Comment by auscaster** [comment]
Can you provide request for bad data?

File: src/store/modules/contacts/getters.js (hunk 1 above)
Comment (chillburn) [reply src/store/modules/contacts/getters.js:45]: I'll can add in the other users only if they dont exist via the user_id. At the moment, FITCare only has an issue with the team admin. Once you fix the team/members endpoint, i'll be able to test this properly.

File: src/store/modules/contacts/getters.js [hunk 2]
```diff
@@ -39,12 +39,12 @@ const getters = {
...
 
-  filteredContactsSorted: (state, getters, rootState, rootGetters) => (excludeUserId = 0) => {
-    if (!state.contacts.length) return [];
-
+  filteredContactsSorted: (state, getters, rootState, rootGetters) => {
+    if (!state.contacts.length && !rootState.team.teamMembers.length) return [];
     const contacts = state.contacts.filter(x => x.friend_id !== rootState.user.id && ['active', 'accepted'].includes(x.status));
-
-    return getters.contactsSortedGroupedByName(contacts);
+    const teamMembers = rootState.team.teamMembers.filter(x => contacts.map(x => x.friend_id).includes(x.user_id) === false && x.user_id !== rootState.user.id);
```
Comment (auscaster) [inline src/store/modules/contacts/getters.js:45]: `contacts.map(x => x.friend_id)` needs to be set outside of loop scope

**Pull request merged**